pom.xml

```diff
@@ -20,7 +20,7 @@
   <groupId>co.cask.http</groupId>
   <artifactId>netty-http</artifactId>
   <version>0.12.0-SNAPSHOT</version>
-  <packaging>jar</packaging>
+  <packaging>bundle</packaging>
```

Comment:
I am using it as a Maven dependency as follows:

<dependency>
            <groupId>co.cask.http</groupId>
            <artifactId>netty-http</artifactId>
            <version>${netty-http.version}</version>
        </dependency>

We don't specify whether it is a jar or bundle when we require the dependency. Also the bundle plugin creates a normal jar file and it just adds some additional headers to the MANIFEST.MF file, which makes it possible to deploy the jar into an OSGi runtime. Existing Maven projects which simply refer to this dependency, there will be no issue since the jar file created by the bundle plugin will be available in the Maven repo.